python/cugraph-dgl/cugraph_dgl/dataloading/dataloader.py

```diff
@@ -54,12 +54,13 @@ def __init__(
         batch_size: int = 1024,
         drop_last: bool = False,
```

Comment:
@tingyu66 , Can we also change `seeds_per_call` to `100_000` to make a better default based on your testing ?

Reply:
Should we change it to the default value of BulkSampler: `200_000`? After our call the other day, I tested a wide range of `seeds_per_call` values and none of the runs threw a OOM error.

Reply:
Interesting,  if it just works we can probably set the default  to None and let upstream handle it  ? What do you think, any default which is reasonable and just works is fine by me.

Reply:
I updated the value to `200_000` in [45f93f2](https://github.com/rapidsai/cugraph/pull/3887/commits/45f93f27611721d3fbc3fa8d845958c71500350a) to align with BulkSampler. Did not set to None to avoid the extra step of handling None case.